### ISSUE TITLE
Document Linux-only target?

### ISSUE THREAD

**Issue by mlinksva** [socs-dev-env/dosocs2#93]
Can't be installed on Mac/OS X easily:

- `scripts/install-nomos.sh` expects `/usr/include/glib-2.0` but homebrew puts it in `/usr/local/include/glib-2.0` (easy enough)
- installation obtains "make nomos standalone agent install /usr/bin/install -D nomossa/usr/local/share/fossology/nomos/agent/nomossa /usr/bin/install: illegal option -- D") and it appears from https://github.com/fossology/fossology/issues/14 and https://github.com/fossology/fossology/issues/363 and the FLOSSology README that OS X isn't supported/only Linux is.

Maybe the DoSOSCv2 README could state something like "Works on platforms FOSSology is available for, currently only Linux"?

**Comment by ghost** [comment]
@mlinksva On Windows and OS X, DoSOCSv2 can be run with Docker. 
- Here  is the URL to docker image. https://hub.docker.com/r/ushan89/dosocsv2/ 
- Docker image is currently awaiting a merge into master (https://github.com/DoSOCSv2/DoSOCSv2/pull/88)
- Once the pull request is done, I will update the documentation for Docker instructions. Nomos is not being compiled for Windows OR OS X at this point.
